jina/peapods/pods/compound.py

```diff
@@ -119,7 +118,6 @@ def start(self) -> 'CompoundPod':
                 self.head_pea = Pea(head_args)
                 self._enter_pea(self.head_pea)
                 for _args in self.replicas_args:
```

Comment:
this is important to be kept. For the inner replica pod right?

Reply:
It is kept. I just moved it to the place, where all args are set. Inside `_set_replica_args`. I prefer to have only one place in the code, where args are set.